scripts/prestart.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# FIXME: this is an interim solution until paritytech/substrate-light-ui#10 gets merged
+# which is currently blocked by facebook/create-react-app#5645
+
+echo Prebuilding Dependent Apps...
+
+echo Building ui-components...
+pushd './packages/ui-components'
```

Comment:
Actually, you only need a pushd in the beginning to save the working dir, `cd` as much as you want in the middle, and popd at the end to restore.

Something like https://github.com/paritytech/substrate/blob/db46682d1ca4c684761785a63881e0a82cc6d7ef/scripts/build.sh#L12-L13